Mathlib/LinearAlgebra/Matrix/Trace.lean

```diff
@@ -42,6 +42,10 @@ def trace (A : Matrix n n R) : R :=
   ∑ i, diag A i
 #align matrix.trace Matrix.trace
 
+lemma trace_diagonal {o} [Fintype o] [DecidableEq o] (d : o → R) :
+    trace (diagonal d) = (Finset.univ : Finset o).sum d := by
```

Comment:
```suggestion
    trace (diagonal d) = ∑ i, d i := by
```
I *think* this is better, but if you have a use case where that is annoying feel free to ignore.

Reply:
I am happy to go with this spelling, thanks!